src/components/displayDocuments/documentCard.tsx

```diff
@@ -28,13 +29,25 @@ interface DocumentCardProps {
 }
 
 export const DocumentCard: React.FunctionComponent<DocumentCardProps> = props => {
+  const [qr, setQr] = useState(false);
+
   return (
     <a href={props.url}>
       <CardContainer className="flex justify-center rounded">
         <div className="flex-column">
           <div className="relative">
-            <DocumentContainer className="rounded" src={props.image} alt="document" />
-            <QrCode />
+            {!qr ? (
+              <DocumentContainer className="rounded" src={props.image} alt="document" />
+            ) : (
+              <RenderQrCode url={props.url} />
+            )}
+            <QrCodeButton
```

Comment:
should change this button when displaying the qr cde

Reply:
what should the button become?

Reply:
an icon with a document instead ? or sth like this ? dfinitely not the qr code anymore